docs/quest_template_addon.md

```diff
@@ -10,41 +10,24 @@ Contains extra definitions like linking quests, dependencies and requirements fo
-| [SpecialFlags][17]          | TINYINT   | UNSIGNED   |     | NO   |         |       |                                       |
-
-[1]: #id
-[2]: #maxlevel
-[3]: #allowableclasses
-[4]: #sourcespellid
-[5]: #prevquestid
-[6]: #nextquestid
-[7]: #exclusivegroup
-[8]: #rewardmailtemplateid
-[9]: #rewardmaildelay
-[10]: #requiredskillid
-[11]: #requiredskillpoints
-[12]: #requiredminrepfaction
-[13]: #requiredmaxrepfaction
-[14]: #requiredminrepvalue
-[15]: #requiredmaxrepvalue
-[16]: #provideditemcount
-[17]: #specialflags
+| [ID](#id)                    | MEDIUMINT | UNSIGNED   | PRI | NO   |         |       | Unique ID linked to quest_template.ID |
+| [MaxLevel](#maxlevel)               | TINYINT   | UNSIGNED   |     | NO   |         |       |                                       |
+| [AllowableClasses](#allowableclasses)       | INT       | UNSIGNED   |     | NO   |         |       |                                       |
+| [SourceSpellID](#sourcespellid)          | MEDIUMINT | UNSIGNED   |     | NO   |         |       |                                       |
+| [PrevQuestID](#prevquestid)            | MEDIUMINT |            |     | NO   |         |       |                                       |
+| [NextQuestID](#nextquestid)            | MEDIUMINT |            |     | NO   |         |       |                                       |
+| [ExclusiveGroup](#exclusivegroup)         | MEDIUMINT |            |     | NO   |         |       |                                       |
+| [RewardMailTemplateID](#rewardmailtemplateid)   | MEDIUMINT | UNSIGNED   |     | NO   |         |       |                                       |
+| [RewardMailDelay](#rewardmaildelay)        | INT       | UNSIGNED   |     | NO   |         |       |                                       |
+| [RequiredSkillID](#requiredskillid)       | SMALLINT  | UNSIGNED   |     | NO   |         |       |                                       |
+| [RequiredSkillPoints](#requiredskillpoints)   | SMALLINT  | UNSIGNED   |     | NO   |         |       |                                       |
+| [RequiredMinRepFaction](#requiredminrepfaction) | SMALLINT  | UNSIGNED   |     | NO   |         |       |                                       |
+| [RequiredMaxRepFaction](#requiredmaxrepfaction) | SMALLINT  | UNSIGNED   |     | NO   |         |       |                                       |
+| [RequiredMinRepValue](#requiredminrepvalue)   | MEDIUMINT |            |     | NO   |         |       |                                       |
+| [RequiredMaxRepValue](#requiredmaxrepvalue)   | MEDIUMINT |            |     | NO   |         |       |                                       |
+| [ProvidedItemCount](#provideditemcount)     | TINYINT   | UNSIGNED   |     | NO   |         |       |                                       |
+| [SpecialFlags](#specialflags)          | TINYINT   | UNSIGNED   |     | NO   |         |       |                                       |
```

Comment:
if you use an auto formatter this text will have a better shape

Reply:
I'm using Visual Studio Code for ending the md files, is there a specific functionality within it or it's like an extension to it?

Reply:
yep